src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2658,3 +2659,26 @@ void java_bytecode_convert_method(
 
   java_bytecode_convert_method(class_symbol, method);
 }
+
+const bool java_bytecode_convert_methodt::is_method_inherited(
+    const irep_idt &classname, const irep_idt &methodname) const
+{
+  class_hierarchyt ch;
+  ch(symbol_table);
+  const auto &parents=ch.get_parents_trans(classname);
+  for(const auto &parent : parents)
+  {
+    forall_symbols(it, symbol_table.symbols)
```

Comment:
Instead of searching the symbol table, construct the precise name you'd be looking for (using the same logic as `remove_virtual_functionst::get_method` to build the symbol name) and check for its existence without iterating.